dashboard/views.py

```diff
@@ -578,7 +578,8 @@ def find_min_week(course_id):
 
 
 def find_current_week(row):
-    current_date = timezone.now()
```

Comment:
I looks like the [recommended](https://docs.djangoproject.com/en/dev/topics/i18n/timezones/#usage) way to do this is
`timezone.localtime(timezone.now())`

Reply:
Done